sdk/core/core/inc/az_http_response.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_HTTP_RESPONSE_H
+#define AZ_HTTP_RESPONSE_H
+
+#include <az_mut_span.h>
+
+#include <_az_cfg_prefix.h>
+
+typedef struct {
+  az_mut_span value;
+} az_http_response;
```

Comment:
Please consider the https://github.com/Azure/azure-sdk-for-c/issues/219 problem

Reply:
I believe standar http response RFC7230 adds \0 at the end of body.  Will make sure later